doc/exploring-cheriot-rtos.md

```diff
@@ -29,21 +29,22 @@ For examples of using these rules, look at a root `xmake.lua` file, such as [`ex
 ## Building an upstream CHERIoT RTOS example
 
 The examples in [`cheriot-rtos/examples`](../cheriot-rtos/examples) provide a nice tour of the different ways compartments can interact.
-These can be built by pointing xmake to the example one wants to build, as shown below.
+These can be built by pointing xmake to the example one wants to build, as shown below:
 
 ```sh
 # Run from the root of the sonata-software repository
 xmake -P cheriot-rtos/examples/05.sealing/
 ```
 
-### Where's my UF2
+### Where's my UF2?
 
 If you've followed the '[running software on the FPGA]' guide, you'll expect UF2 files as part of the build artefacts but these aren't automatically created in the [`cheriot-rtos`][] repository.
-Thankfully, this repository includes a `./scripts/elf-to-uf2.sh` script that converts an ELF into a firmware a UF2 file.
-
-[running software on the FPGA]: ./guide/running-software.md#running-on-the-sonata-fpga
+You can generate it using the following commands:
 
 ```sh
 xmake -P cheriot-rtos/examples/05.sealing/
-./scripts/elf-to-uf2.sh build/cheriot/cheriot/release/sealing
+llvm-strip build/cheriot/cheriot/release/sealing -o sealing_stripped.elf
+uf2conv -b 0x0000 -f 0x6ce29e60 sealing_stripped.elf -co sealing.elf.uf2
```

Comment:
This script was coming in the next PR; should I still include it or is this preferred?

Reply:
Adding that script would be great. Can you change this back in the PR where you add it?

Reply:
Actually I'll just drop this commit.